linera-views/src/reentrant_collection_view.rs

```diff
@@ -509,50 +534,9 @@ where
                 let Update::Set(view) = entry.into_mut() else {
                     unreachable!()
                 };
```

Comment:
Why not just `get_mut()` instead of `entry()` and `into_mut()`?

```rust
let Some(Update::Set(view)) = updates.get_mut(&short_key) else {
    unreachable!()
};
```

Reply:
Thank you.